src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationIntegrationTest.java

```diff
@@ -226,4 +226,29 @@ private void validateUsers(Map<String, Object> response, int expectedUserCount,
             }
         });
     }
+
+    @Test
+    public void can_retrieve_users_when_false_should_return_status_200_without_roles() {
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(createAndActivateOrganisation(), "True", hmctsAdmin,"false");
+        validateUsers(response, 3, false);
+    }
+
+    @Test
+    public void can_retrieve_users_when_true_should_return_status_200_with_roles() {
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(createAndActivateOrganisation(), "True", hmctsAdmin,"true");
+        validateUsers(response, 3, true);
+    }
+
+    @Test
+    public void can_retrieve_users_when_default_should_return_status_200_with_roles() {
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(createAndActivateOrganisation(), "True", hmctsAdmin,null);
+        validateUsers(response, 3, true);
+    }
+
+    @Test
+    public void cannot_retrieve_users_when_invalid_user_roles_should_return_status_403() {
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(createAndActivateOrganisation(), "True", "InvalidRole",null);
+        assertThat(response.get("http_status")).isEqualTo("403");
+    }
+
```

Comment:
Please add an integration test that passes in an invalid value for `returnRoles` param and should return a 400 Status Code

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/612/commits/acbe4881fa2e5d46be921bc81e0a93e8f8bf9f4f